examples/protocol_specification_ex/fipa.yaml

```diff
@@ -5,16 +5,16 @@ license: Apache 2.0
 description: 'A protocol for FIPA ACL.'
 speech_acts:
   cfp:
-    query: Query
+    query: ct:Query
   propose:
-    proposal: Proposal
+    proposal: ct:Proposal
     price: pt:float
   accept_w_inform:
-    info: pt:dict[pt:str, AcceptInfo]
+    info: pt:dict[pt:str, ct:AcceptInfo]
```

Comment:
For this protocol, I think we could have `ct:Info` for all three as this is what we actually need.

Reply:
Done